src/components/Explore/Explore.tsx

```diff
@@ -41,9 +45,16 @@ const Explore: React.FunctionComponent<{ projection: Projection }> = ({
       </Styles.ChartControlsContainer>
       <Styles.ChartContainer>
         <ParentSize>
-          {({ width }) => (
-            <ExploreChart series={series} width={width} height={400} />
-          )}
+          {({ width }) =>
+            width < 10 ? null : (
```

Comment:
It's to avoid rendering the chart before the width is available (there is a moment when that is `0`), but I think there is a better way, will clean it up or put a comment.